<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>442</width>
    <height>358</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>iOS Reader Applications</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/images/lt.png</normaloff>:/images/lt.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0">
    <widget class="QGroupBox" name="debug_gb">
     <property name="title">
      <string>Debug options</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="0" column="0">
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <widget class="QCheckBox" name="debug_plugin">
          <property name="text">
           <string>Enable debug logging</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QGroupBox" name="general_gb">
       <property name="title">
        <string>General options</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="2">
         <widget class="QToolButton" name="wizard_tb">
          <property name="toolTip">
           <string>Add syncing service</string>
          </property>
          <property name="text">
           <string>w</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="3">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>&lt;html&gt;
&lt;head&gt;
&lt;style&gt;
h4  {font-style: italic; margin:0; padding:0;}
ul {margin-top:0; padding-bottom: 1em; padding-top:0;}
&lt;/style&gt;
&lt;/head&gt;
&lt;body&gt;
&lt;h4&gt;To enable a syncing service:&lt;/h4&gt;
&lt;ul&gt;
&lt;li&gt;Click the Wizard button&lt;/li&gt;
&lt;li&gt;Follow the steps&lt;/li&gt;
&lt;li&gt;Restart calibre&lt;/li&gt;
&lt;/ul&gt;
&lt;h4&gt;To forget a syncing service:&lt;/h4&gt;
&lt;ul&gt;
&lt;li&gt;Click the Delete button&lt;/li&gt;
&lt;/ul&gt;
&lt;h4&gt;To disable SyncMan:&lt;/h4&gt;
&lt;ul&gt;
&lt;li&gt;Select a blank entry from the dropdown list&lt;/li&gt;
&lt;/ul&gt;
&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <widget class="QComboBox" name="sync_apps">
            <property name="toolTip">
             <string>Syncing service to manage</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="0" column="1">
         <widget class="QToolButton" name="forget_tb">
          <property name="toolTip">
           <string>Forget this syncing service</string>
          </property>
          <property name="text">
           <string>x</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
